internal/config/operations_test.go

```diff
@@ -0,0 +1,135 @@
+	}
+	if op.AdditionalProperties[ServiceName] != "jaeger" {
+		t.Errorf("Expected %v for %v additional property but got %v", "jaeger", ServiceName, op.AdditionalProperties[ServiceName])
+	}
+	if op.AdditionalProperties[ServicePatchFile] != "file://templates/oam/patches/service-loadbalancer.json" {
+		t.Errorf("Expected %v for %v additional property but got %v", "file://templates/oam/patches/service-loadbalancer.json", ServicePatchFile, op.AdditionalProperties[ServicePatchFile])
+	}
+	if op.AdditionalProperties[HelmChartURL] != "https://helm.linkerd.io/stable/linkerd-jaeger-30.4.5.tgz" {
+		t.Errorf("Expected %v for %v additional property but got %v", "https://helm.linkerd.io/stable/linkerd-jaeger-30.4.5.tgz", HelmChartURL, op.AdditionalProperties[HelmChartURL])
+	}
+
+	// Test for MultiClusterAddon
+	op, exists = dev[MultiClusterAddon]
+	if !exists {
+		t.Errorf("Operation %v not found in operations map", MultiClusterAddon)
+	}
+	if op.Type != int32(meshes.OpCategory_CONFIGURE) {
+		t.Errorf("Expected operation type %v but got %v", int32(meshes.OpCategory_CONFIGURE), op.Type)
+	}
+	if op.Description != "Add-on: Multi-cluster" {
+		t.Errorf("Expected operation description %v but got %v", "Add-on: Multi-cluster", op.Description)
+	}
+	if len(op.AdditionalProperties) != 3 {
+		t.Errorf("Expected 3 additional properties but got %v properties", len(op.AdditionalProperties))
+	}
+	if op.AdditionalProperties[ServiceName] != "linkerd-gateway" {
+		t.Errorf("Expected %v for %v additional property but got %v", "linkerd-gateway", ServiceName, op.AdditionalProperties[ServiceName])
+	}
+	if op.AdditionalProperties[ServicePatchFile] != "file://templates/oam/patches/service-loadbalancer.json" {
+		t.Errorf("Expected %v for %v additional property but got %v", "file://templates/oam/patches/service-loadbalancer.json", ServicePatchFile, op.AdditionalProperties[ServicePatchFile])
+	}
+	if op.AdditionalProperties[HelmChartURL] != "https://helm.linkerd.io/stable/linkerd-multicluster-2.10.2.tgz" {
+		t.Errorf("Expected %v for %v additional property but got %v", "https://helm.linkerd.io/stable/linkerd-multicluster-2.10.2.tgz", HelmChartURL, op.AdditionalProperties[HelmChartURL])
+	}
+
+	// Test for SMIAddon
+	op, exists = dev[SMIAddon]
+	if !exists {
+		t.Errorf("Operation %v not found in operations map", SMIAddon)
+	}
+	if op.Type != int32(meshes.OpCategory_CONFIGURE) {
+		t.Errorf("Expected operation type %v but got %v", int32(meshes.OpCategory_CONFIGURE), op.Type)
+	}
+	if op.Description != "Add-on: SMI Addon" {
+		t.Errorf("Expected operation description %v but got %v", "Add-on: SMI Addon", op.Description)
+	}
+	if len(op.AdditionalProperties) != 1 {
+		t.Errorf("Expected 1 additional properties but got %v properties", len(op.AdditionalProperties))
+	}
+	if op.AdditionalProperties[HelmChartURL] != "https://github.com/linkerd/linkerd-smi/releases/download/v0.1.0/linkerd-smi-0.1.0.tgz" {
```

Comment:
In my opinion, I believe we need to update these addresses of service mesh manually. So, it is good we add them to the `config.go` as a variable. And we can use it in both `operation.go` and `operation_test.go` but we should be simple. So, this is not related to current PR.

Reply:
So shall I remove line 132?

Reply:
Keep it in this PR. I believe it is ok. We can merge this first and append another PR for these changes.